python/mxnet/gluon/model_zoo/vision/mobilenet.py

```diff
@@ -53,15 +102,18 @@ class MobileNet(HybridBlock):
     classes : int, default 1000
         Number of classes for the output layer.
     """
+
     def __init__(self, multiplier=1.0, classes=1000, **kwargs):
         super(MobileNet, self).__init__(**kwargs)
         with self.name_scope():
             self.features = nn.HybridSequential(prefix='')
             with self.features.name_scope():
-                _add_conv(self.features, channels=int(32*multiplier), kernel=3, pad=1, stride=2)
-                dw_channels = [int(x*multiplier) for x in [32, 64]+[128]*2+[256]*2+[512]*6+[1024]]
-                channels = [int(x*multiplier) for x in [64]+[128]*2+[256]*2+[512]*6+[1024]*2]
+                _add_conv(self.features, channels=int(32 * multiplier), kernel=3, pad=1, stride=2)
+                # pylint: disable= line-too-long
+                dw_channels = [int(x * multiplier) for x in [32, 64] + [128] * 2 + [256] * 2 + [512] * 6 + [1024]]
+                channels = [int(x * multiplier) for x in [64] + [128] * 2 + [256] * 2 + [512] * 6 + [1024] * 2]
                 strides = [1, 2] * 3 + [1] * 5 + [2, 1]
+                # pylint: disable= invalid-name
```

Comment:
This is mobilenet v1 code, add this to pass the style check.
autopep8 made those changes.

Reply:
please use `make pylint` as the standard for checking code style. we have a 100 char-per-line limit.